src/linked_list.rs

```diff
@@ -234,6 +234,27 @@ impl<'a, A: Adapter<Link = Link>> Cursor<'a, A> {
         }
     }
 
+    /// Clones and returns the pointer that points to the element that the
+    /// cursor is referencing.
+    ///
+    /// This returns None if the cursor is currently pointing to the null
```

Comment:
Use backticks around `None`.

Reply:
This was copied from the `get` method. I updated both methods to use backticks now